model-optimizer/extensions/front/onnx/softmax_ext.py

```diff
@@ -16,7 +16,8 @@
 
 from mo.front.extractor import FrontExtractorOp
 from mo.front.onnx.extractors.utils import onnx_attr
-from mo.ops.softmax import LogSoftmaxONNX, SoftmaxONNX
+from mo.ops.softmax import SoftmaxONNX
+from mo.ops.log_softmax import LogSoftmaxONNX
 
 
 class SoftmaxExtractor(FrontExtractorOp):
```

Comment:
I would also split extractors for SoftMax and LogSoftMax.

Reply:
I would consider this like a "nit" comment. So if there will be no comments from @ilyachur or @GlebKazantaev  I would merge the PR.